app/src/main/java/com/babylon/wallet/android/domain/usecases/transaction/PollPreAuthorizationStatusUseCase.kt

```diff
@@ -0,0 +1,41 @@
+package com.babylon.wallet.android.domain.usecases.transaction
+
+import com.babylon.wallet.android.data.repository.PreAuthorizationStatusData
+import com.babylon.wallet.android.di.coroutines.IoDispatcher
+import com.radixdlt.sargon.DappToWalletInteractionSubintentExpiration
+import com.radixdlt.sargon.PreAuthorizationStatus
+import com.radixdlt.sargon.SubintentHash
+import com.radixdlt.sargon.os.SargonOsManager
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.withContext
+import javax.inject.Inject
+
+class PollPreAuthorizationStatusUseCase @Inject constructor(
```

Comment:
I was thinking maybe we considered renaming the use cases and the repository methods to `Get` instead of `Poll`. It looks like we have now an API that returns a success or an error and we don't, as hosts, poll for anything. Very minor in my opinion.